src/components/CourseInformation/RelatedCourses/RelatedCourses.js

```diff
@@ -0,0 +1,26 @@
+import ExperienceCard from "../../ExperienceCard/ExperienceCard";
+
+const RelatedCourses = (props) => {
+  // Expects an array of courses
+  const courses = props.data;
+
+
+  // If the data is not there 
+  if (!courses) {
+    return <div>Loading...</div>;
```

Comment:
The loading should take place in the parent component/wherever the API call is being made. If it's loading or errored out, don't render this component at all, else render this component.

Reply:
I agree. We will remove/refactor that